src/remote/watcher.js

```diff
@@ -56,6 +71,9 @@ export default class RemoteWatcher {
       await this.pouch.setRemoteSeqAsync(changes.last_seq)
       log.lineBreak()
     } catch (err) {
+      if (err.message === 'Client has been revoked') {
```

Comment:
Yeah, I'm discovering that everything error-related in js is a lot of fun...

I was thinking of [es5-style](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error#Custom_Error_Types).

Regarding the *incorrect lineNumber and fileName* issue, I suppose there's a way to make it work on node since some modules seems to do so, e.g. node-fetch:

https://github.com/bitinn/node-fetch/blob/master/src/fetch-error.js

This makes me think that [my own implementation of `DirectoryNotFound`](https://github.com/cozy-labs/cozy-desktop/blob/master/src/remote/cozy.js#L21-L23) is not really production-ready...